src/components/healthtips.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { Container, Row } from 'bootstrap-4-react';
+import Navbar from '../components/navbar';
+import './style.css';
+
+const content = [
+  {
+    aboutNumber: 'EAT HEALTHY',
+    aboutContent:
+      'Eating well is vital for healthy living, but it’s not something that students are always good at. There’s a reason for all those cold beans and instant noodle stereotypes, after all. This is changing, though. But the most important thing is making time to eat proper, healthy meals on a regular basis.',
+  },
+  {
+    aboutNumber: 'GOOD SLEEP',
+    aboutContent:
+      'Sleep is just as important as diet and exercise for a healthy lifestyle. Ideally, you should get between seven and nine hours every night. If this isn’t currently the case, cutting back on caffeine after 3pm and avoiding illuminated screens (phones, tablets, etc.) before bed could help.',
+  },
+  {
+    aboutNumber: 'HEALTHY MIND',
+    aboutContent:
+      'There are simple ways you can help stay health:-Being aware of your own mental health and stress levels. Knowing your stress triggers. Taking time out to relax now and then. Meditation and some excercises.',
+  },
+];
+
+class Card extends Component {
+  render() {
+    return (
+      <div className='box-container'>
+        <div className='box-item'>
+          <div className='flip-box'>
+            <div
+              className='flip-box-front text-center'
+              style={{
+                backgroundImage: `url(https://i.pinimg.com/originals/7c/05/50/7c05508149920eba2d934c19ff83cbba.jpg)`,
+              }}
+            >
+              <div className='inner color-white'>
+                <br />
+                <br />
+                <h3
+                  className='flip-box-header'
+                  style={{ textTransform: 'uppercase' }}
+                >
+                  {this.props.content.aboutNumber}
+                </h3>
+                <br />
+                <img
+                  src='https://s25.postimg.cc/65hsttv9b/cta-arrow.png'
+                  alt=''
+                  className='flip-box-img'
+                />
+              </div>
+            </div>
+            <div
+              className='flip-box-back text-center'
+              style={{
+                backgroundImage: `url(https://i.pinimg.com/736x/5e/e2/db/5ee2db0b6b3098b78812712d137c102d.jpg)`,
+              }}
+            >
+              <div className='inner color-white'>
+                {this.props.content.aboutContent}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+class healthtips extends Component {
+  render() {
+    return (
+      <>
+        <div>
+          <Navbar />
+          <br />
+          <br />
+          <br />
+          <Container
+            style={{
```

Comment:
Please see [this](https://github.com/Namanl2001/MERN-Gurujii-dev/blob/86d5ed01f0f2657a0d94d5e44b6426802c930eff/src/components/style.css#L801).

If `5em` looks more good in your changes then you can make a new styling in `style.css` and use it here.

Reply:
![image](https://user-images.githubusercontent.com/65587816/119354777-15c76b00-bcc2-11eb-8ad2-7fad2ec4d4e2.png)
I think 2rem also looks fine. Updated the same.